users.json

```diff
@@ -0,0 +1,167 @@
+[
```

Comment:
Chat GPT can generate such dummy data, maybe consider to use it in this particular case.